components/predict-form/QuestionTypeSelect.tsx

```diff
@@ -15,9 +15,7 @@ export function QuestionTypeSelect({
       <button
         className={clsx(
           "btn",
-          questionType === QuestionType.BINARY
-            ? "btn-primary"
-            : "text-neutral-500",
+          questionType === QuestionType.BINARY ? "" : "btn-ghost",
```

Comment:
nit: I'd use
`questionType !== QuestionType.BINARY && "btn-ghost"`

here and below